### PR TITLE
PF-469 Add a test that a duplicate controlled AI notebook instance cannot be created.

### PR DESCRIPTION
**Pull request by wchamber**

### CODE REVIEW

File: src/main/java/bio/terra/workspace/service/job/JobService.java [hunk 1]
```diff
@@ -52,6 +45,12 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
+import org.apache.commons.lang3.StringUtils;
```
Comment (ddietterich) [inline src/main/java/bio/terra/workspace/service/job/JobService.java:48]: I wonder if we can get the IntelliJ plugin and the spotless to agree on where the imports should go :(

**Review by ddietterich — APPROVED** [review]
LGTM

**Review by zloery — APPROVED** [review]
